iotedgedev/deploymentmanifest.py

```diff
@@ -10,12 +10,13 @@
 
 
 class DeploymentManifest:
-    def __init__(self, envvars, output, path, is_template):
+    def __init__(self, envvars, output, utility, path, is_template):
+        self.utility = utility
         self.output = output
         try:
             self.path = path
             self.is_template = is_template
-            self.json = json.load(open(path))
+            self.json = json.loads(os.path.expandvars(self.utility.get_file_contents(path)))
```

Comment:
Let's put this expand/get command in utility.

Reply:
Moved. Thanks.